addons/netfox/network-time.gd

```diff
@@ -362,10 +364,8 @@ func _process(delta):
 	_process_delta = delta
```

Comment:
Pin the current time for smooth interpolation

```suggestion
	_process_delta = delta
	_current_tick = _get_os_time()
```

Reply:
Did something similar, let me know what you think

Reply:
Nice! Didn't even consider pausing.